core/CMakeLists.txt

```diff
@@ -14,23 +14,26 @@ endif()
 
 # We need tinyxml2 for the camera definition parsing.
 if(APPLE)
-    # We install the tinyxml2 library manually for macOS and iOS.
-    if (CMAKE_BUILD_TYPE STREQUAL "Debug")
-        # Need to remove that d again.
-        install(FILES ${CMAKE_CURRENT_BINARY_DIR}/third_party/tinyxml2/libtinyxml2d.5.0.1.dylib
-            DESTINATION ${lib_path}
-            #RENAME libtinyxml2.dylib
-        )
-    else()
-        install(FILES ${CMAKE_CURRENT_BINARY_DIR}/third_party/tinyxml2/libtinyxml2.5.0.1.dylib
-            DESTINATION ${lib_path}
-            #RENAME libtinyxml2.dylib
-        )
-    endif()
+    if(NOT IOS)
```

Comment:
You could just write `if (APPLE AND NOT IOS)`, I think.

Reply:
Nope, because there is an `else()` at the end of the tinyxml2 block, that would then take linux and iOS :)

Reply:
Ah, you're right.